services/movie.py

```diff
@@ -0,0 +1,31 @@
+import init_django_orm  # noqa: F401
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> set[Movie]:

```

Comment:
Optional[list[int]] because default parameters is None

Reply:
Also, check in other places if default parameter is None it already Optional[None]